CMake/global_config.cmake

```diff
@@ -23,6 +23,12 @@ macro(global_set_flags)
 
     add_definitions(-DELPP_THREAD_SAFE)
     add_definitions(-DELPP_NO_DEFAULT_LOG_FILE)
+    
+    #Performance improvement with Ubuntu 18/20
+    if(UNIX AND (NOT ANDROID_NDK_TOOLCHAIN_INCLUDED))
```

Comment:
I suggest this be made into an option (on by default with these same conditions) that can be disabled if needed

Reply:
Please detail what you mean here. The flag can still be disabled in the code where it is checked.
What other option would you wish to have?

Reply:
I mean that, when the user uses CMake, they can see the option and turn it off if they want to -- and not by changing the code

Reply:
flag added